ossdbtoolsservice/parsers/mysql_datatypes.py

```diff
@@ -0,0 +1,42 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+''' mysql data types. We need to make sure all the data types are defined with name starting with DATATYPE_ as we
+pull them into the array of system datatypes '''
+
+''' null Type '''
+DATATYPE_NULL = 'NULL'
+
+''' Numeric Types '''
+DATATYPE_TINYINT = 'tinyint'
+DATATYPE_SMALLINT = 'smallint'
+DATATYPE_MEDIUMINT = 'mediumint'
+DATATYPE_INTEGER = 'int'
+DATATYPE_BIGINT = 'bigint'
+DATATYPE_DECIMAL = 'decimal'
+DATATYPE_NUMERIC = 'numeric'
+DATATYPE_FLOAT = 'float'
+DATATYPE_DOUBLE = 'double'
+DATATYPE_BIT = 'bit'
+
+''' Character Types '''
+DATATYPE_CHAR = 'char'
+DATATYPE_VARCHAR = 'varchar'
+DATATYPE_BINARY = 'binary'
+DATATYPE_VARBINARY = 'varbinary'
+DATATYPE_BLOB = 'blob'
```

Comment:
Additional datatypes are also available.
Example : TINYBLOB, etc

Reply:
BLOB datatypes are not editable. I have added parser for JSON datatype in the latest commit.